src/main/java/de/danielbechler/diff/access/PropertyAwareAccessor.java

```diff
@@ -29,4 +29,8 @@
 	Set<Annotation> getReadMethodAnnotations();
 
 	<T extends Annotation> T getReadMethodAnnotation(Class<T> annotationClass);
+
+   <T extends Annotation> T getAnnotation(Class<T> annotationClass);
+
+   int getFieldModifiers();
```

Comment:
Awesome! That's great to hear! I'll cherry-pick the PrimitiveArrayDiffer from this PR and see if I can make it a default, if you don't mind?